docs/source/gettingstarted.rst

```diff
@@ -191,6 +191,19 @@ They are:
 Any combination of these arguments can be given. Omitting them all is fine too.
 Their behavior is documented in detail below.
 
+``disable_y_normalization``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specifies whether normalization for the y axis will be disabled. By default, the y axis readings will be normalized, but if disable_y_normilization is set to true then the y axis will not be normalized.
+A few examples of using this argument::
+
+    # set analysis to use y normalization.
+    write_xlsx(r) or write_xlsx(r, disable_y_normalization=False)
```

Comment:
`or` is a real python key word so I would maybe split these two examples onto their own lines